chat-widget/src/components/webchatcontainerstateful/webchatcontroller/middlewares/renderingmiddlewares/messageSequenceIdOverrideMiddleware.tsx

```diff
@@ -0,0 +1,66 @@
+import { IWebChatAction } from "../../../interfaces/IWebChatAction";
```

Comment:
Can you please verify behavior after C2 disconnects (both close browser and network disconnect? C1 sending messages during C2 disconnect

Reply:
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/981914/99a70f3b-0117-4c7c-8d87-c01ecf8b2397)

No errors thrown when offline ,

Reply:
Could you verify after C2 disconnects, C1 sent messages order are also preserved after C2 reconnects?

Reply:
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/981914/2c9be020-43f3-4f09-8ace-ad90a621dec0)
![image](https://github.com/microsoft/omnichannel-chat-widget/assets/981914/28722f8b-44ee-4acd-86e1-2b784bf31ff6)